Makefile

```diff
@@ -293,7 +293,9 @@ deploy-onprem-for-subsystem:
 	export DUMMY_IGNITION="true" && $(MAKE) deploy-onprem
 
 deploy-on-openshift-ci:
+ifeq ("$(wildcard $(shell which kubectl))","")
```

Comment:
IMO you can remove 
```
ln -s $(shell which oc) $(shell dirname $(shell which oc))/kubectl
```

Reply:
I'm looking to test with and without the custom build root image so I still need it for that.